# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/TestMediaDataEncoder.cpp
# Commit: ea4d0db67f89
# Full Hash: ea4d0db67f899714106e6f4ffcc78ebe8feaf466
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - Implement plumbing for SVC. r=chunmin,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D196473
# ==============================================================================

diff -r f0ac27321cf8 -r ea4d0db67f89 dom/media/gtest/TestMediaDataEncoder.cpp
--- a/dom/media/gtest/TestMediaDataEncoder.cpp	Fri Dec 22 21:39:18 2023 +0000
+++ b/dom/media/gtest/TestMediaDataEncoder.cpp	Fri Dec 22 21:39:18 2023 +0000
@@ -174,7 +174,7 @@
       EncoderConfig(aCodec, gfx::IntSize{aWidth, aHeight}, aUsage, aPixelFormat,
                     aPixelFormat, FRAME_RATE /* FPS */,
                     KEYFRAME_INTERVAL /* keyframe interval */,
-                    BIT_RATE /* bitrate */, BIT_RATE_MODE, pref, aSpecific),
+                    BIT_RATE /* bitrate */, BIT_RATE_MODE, pref, MediaDataEncoder::ScalabilityMode::None, aSpecific),
       taskQueue);
 
   return e.forget();